modules/aioseop_sitemap.php

```diff
@@ -3397,72 +3430,119 @@ private function get_images_from_post( $post ) {
 			}
 
 			if ( isset( $post_thumbnails[ $post->ID ] ) ) {
-				$attachment_url = wp_get_attachment_image_url( $post_thumbnails[ $post->ID ], 'post-thumbnail' );
-				if ( $attachment_url ) {
-					$images[] = $attachment_url;
-				}
+				$post_image_ids[] = intval( $post_thumbnails[ $post->ID ] );
 			}
 
-			$content = $post->post_content;
+			$post_image_ids = array_merge( $post_image_ids, $this->get_gallery_image_ids( $post ) );
 
-			$this->get_gallery_images( $post, $images );
+			$this->get_gallery_images( $post, $post_image_urls );
 
+			// Get image URLs from content.
+			$content  = $post->post_content;
 			$content .= $this->get_content_from_galleries( $content );
-			$this->parse_content_for_images( $content, $images );
+			$this->parse_content_for_images( $content, $post_image_urls );
```

Comment:
@EkoJR can we move this and related methods to common?

Reply:
I thought I had mentioned this before in one of the previous PRs. For this PR, I would have to disagree. I had already added changes to that method, and would be better in a separate PR to reflect the changes accurately.

In other words.
* This PR will reflect changes to the function(s).
* Next PR would just be a refactor to reflect that is was just moved with no modifications; which makes sifting through the changes easier for future reference.

Conceptually, I will agree with you. Just not on this PR; since the commits are likely to be squashed.